src/main/webapp/script.js

```diff
@@ -102,7 +102,7 @@ function createEntertainmentItemCard(entertainmentItem) {
       $('<img class="card-img-top" src="' + entertainmentItem.imageUrl + '">'));
   card.append(
       $('<a class="stretched-link" href="item-page.html?itemId=' +
-        entertainmentItem.uniqueId + '"></a>'))
+        entertainmentItem.uniqueId.value + '"></a>'))
```

Comment:
what happens if this optional is empty?

Reply:
The link would have an empty query string so Eric would not be able to get the Id. Do you think I should add a check before calling this function so that the card item does not get created if it does not have an Id assigned?

Reply:
I added a check to see if the optional is empty in the last commit.

Reply:
Yeah the check looks good :)